abipy/lessons/lesson_helper_functions.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+"""
+reusable functions for lessons
+"""
+
+from __future__ import division, print_function
+
+import sys
+import os
+import shutil
+import abipy.abilab as abilab
+import abipy.data as abidata
+
+
+def help(stream=sys.stdout):
+    """
+    Display the tutorial text.
+    """
+    stream.write(__doc__)
```

Comment:
this won't work because it returns the __doc__ string of lesson_helper_functions.py